src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -168,106 +171,157 @@ The new `window-placement` permission can be queried with the
-  Screen 1 (built-in display):
-  availHeight: 975
-  availLeft: 0
-  availTop: 23
-  availWidth: 1680
-  colorDepth: 30
-  height: 1050
-  id: "0"
-  internal: true
-  left: 0
-  orientation: null
-  pixelDepth: 30
-  primary: true
-  scaleFactor: 2
-  top: 0
-  touchSupport: false
-  width: 1680
-
-  Screen 2 (iPad):
-  availHeight: 1001
-  availLeft: 1680
-  availTop: 23
-  availWidth: 1366
-  colorDepth: 24
-  height: 1024
-  id: "1"
-  internal: false
-  left: 1680
-  orientation: null
-  pixelDepth: 24
-  primary: false
-  scaleFactor: 2
-  top: 0
-  touchSupport: false
-  width: 1366
+/* Output from my MacBook Pro 13″ with the iPad attached:
+{
+  currentScreen: ScreenAdvanced {left: 0, top: 0, isPrimary: true, isInternal: true, devicePixelRatio: 2, …}
+  oncurrentscreenchange: null
+  onscreenschange: null
+  screens: [{
+    // The MacBook Pro
+    availHeight: 969
+    availLeft: 0
+    availTop: 25
+    availWidth: 1680
+    colorDepth: 30
+    devicePixelRatio: 2
+    height: 1050
+    id: ""
+    internal: false
+    isExtended: true
+    isInternal: true
+    isPrimary: true
+    label: ""
+    left: 0
+    onchange: null
+    orientation: ScreenOrientation {angle: 0, type: "landscape-primary", onchange: null}
+    pixelDepth: 30
+    pointerTypes: []
+    primary: false
+    scaleFactor: 2
+    top: 0
+    touchSupport: false
+    width: 1680
+  },
+  {
+    // The iPad
+    availHeight: 999
+    availLeft: 1680
+    availTop: 25
+    availWidth: 1366
+    colorDepth: 24
+    devicePixelRatio: 2
+    height: 1024
+    id: ""
+    internal: false
+    isExtended: true
+    isInternal: false
+    isPrimary: false
+    label: ""
+    left: 1680
+    onchange: null
+    orientation: ScreenOrientation {angle: 0, type: "landscape-primary", onchange: null}
+    pixelDepth: 24
+    pointerTypes: []
+    primary: false
+    scaleFactor: 2
+    top: 0
+    touchSupport: false
+  }]
+}
 */
 ```
 
-Note how the value of `left` for the iPad starts at `1680`, which is exactly the `width` of the
-built-in display. This allows me to determine exactly how the screens are arranged logically (next
-to each other, on top of each other, etc.). There is also data now for each screen to show whether
-it is an `internal` one and whether it is a `primary` one.
-Note that the built-in screen
+Information about the connected screens is available in the `screens` array. Note how the value of
+`left` for the iPad starts at `1680`, which is exactly the `width` of the built-in display. This
+allows me to determine exactly how the screens are arranged logically (next to each other, on top of
+each other, etc.). There is also data now for each screen to show whether it is an `isInternal` one
+and whether it is an `isPrimary` one. Note that the built-in screen
 [is not necessarily the primary screen](https://osxdaily.com/2010/04/27/set-the-primary-display-mac/#:~:text=Click%20on%20the%20Display%20icon,primary%20display%20for%20your%20Mac).
```

Comment:
Isn' necessarily or doesn't have to be? The former implies an element of chance, while the later implies my complete control.

Reply:
"Isn't necessarily". Operating systems may have different heuristics as to which plugged in additional screen they make the primary screen.